assets/css/style.css

```diff
@@ -1 +1 @@
15px;height:15px;border:0;border-right:2px solid #151515;border-bottom:2px solid #151515;-webkit-transition:border 0.3s ease;-o-transition:border 0.3s ease;transition:border 0.3s ease}.carousel__buttons .btn:disabled::before{border-color:#efefef}.carousel__buttons .btn.carousel-next::before{-webkit-transform:rotate(-45deg);-ms-transform:rotate(-45deg);transform:rotate(-45deg)}.carousel__buttons .btn.carousel-prev::before{-webkit-transform:rotate(135deg);-ms-transform:rotate(135deg);transform:rotate(135deg)}.carousel__buttons .btn:hover:not([disabled])::before{border-color:#92c064}.pagination-bullet{width:10px;height:10px;border-radius:50%;background:#efefef;margin:0 5px;cursor:pointer;-webkit-transition:background 0.3s ease;-o-transition:background 0.3s ease;transition:background 0.3s ease}.pagination-bullet:hover:not(.pagination-bullet--active){background:#accf88}.pagination-bullet--active{background:#92c064}@media all and (max-width: 599px){#carousel{margin-top:20px}}#products .content-center{margin-bottom:20px}@media all and (min-width: 600px){#products .content-center{margin-bottom:30px}}.products-list{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row;-webkit-box-align:start;-ms-flex-align:start;align-items:flex-start;-webkit-box-pack:start;-ms-flex-pack:start;justify-content:flex-start;-ms-flex-wrap:wrap;flex-wrap:wrap}.products-list__item{padding:16px 8px;border:1px solid #efefef;border-radius:2px;width:calc(50% - 10px);margin-bottom:40px;text-decoration:none;-webkit-transition:border 0.3s ease;-o-transition:border 0.3s ease;transition:border 0.3s ease;margin-bottom:20px}@media all and (max-width: 599px){.products-list__item:nth-child(2n){margin-left:20px}}.products-list__item:hover{border-color:#accf88}@media all and (min-width: 600px){.products-list__item{padding:16px;width:calc(33.333% - 10px);margin-bottom:40px}.products-list__item:nth-child(2),.products-list__item:nth-child(4n-7){margin-left:15px;margin-right:15px}.products-list__item:first-child{margin-left:0px;margin-right:0px}}.products-list__item h3{font-weight:700;margin-top:15px}.products-list__item p{color:#7e7e7e;margin-top:10px}.products-list__image img{width:100%}.products-list__price{font-weight:700}.products-list__controls{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;-webkit-box-align:start;-ms-flex-align:start;align-items:flex-start;-webkit-box-pack:justify;-ms-flex-pack:justify;justify-content:space-between;-ms-flex-wrap:wrap;flex-wrap:wrap;margin-top:20px}@media all and (min-width: 600px){.products-list__controls{-webkit-box-orient:horizontal;-webkit-box-direction:normal;-ms-flex-direction:row;flex-direction:row;-webkit-box-align:center;-ms-flex-align:center;align-items:center}}@media all and (max-width: 599px){.products-list__controls .btn{margin-top:10px}}
```

Comment:
Nitpick: Ideally this file should not be pushed to repo (same as `node_modules`). 😃

Reply:
Why minified main style.css should not be pushed to repo? :)

Reply:
I'm referring to "ideal world", where the app has step to build the source styles into distributable resource, usually - minified and often a single style file.
Also, as we can see, it's just noise when doing code review. 😃